### DISCUSSION TITLE
啊啊啊一开始我登进去了，过了一个晚上怎么又不行了😭

### DISCUSSION THREAD

**Discussion by JingJing431** [share121/inter-knot#1629] [其他]
![Screenshot_2024_0812_015524](https://github.com/user-attachments/assets/4bd3e9b6-c0c0-40d1-9fc0-4b978aab8738)
![Screenshot_20240812_122911](https://github.com/user-attachments/assets/078707d3-dc03-469d-bc17-acf00f2f6e2d)

**Comment by share121** [comment]
你可以试试 x 浏览器，目前对于手机的支持很玄学。

**Reply by JingJing431** [reply]
谢谢！

**Comment by whsjjj28288** [comment]
你猜我现在在用什么给你们说话？用手机via浏览器，可以装插件，昨天的时候我看那插件还是1.6，今天又变成1.6.1了，刚更新了，就登上了

**Reply by JingJing431** [reply]
我就用的via浏览器啊

**Comment by JingJing431** [comment]
懂了，是我网不好，开流量进去了💦💦
